rts/lra/test/pom.xml

```diff
@@ -196,13 +210,54 @@
               </execution>
             </executions>
           </plugin>
+
+          <!-- Makes sure that the two Wildfly instances use different standalone XML configurations -->
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>${version.antrun.plugin}</version>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <phase>generate-test-resources</phase>
+                <configuration>
+                  <target>
+                    <copy file="${env.JBOSS_HOME}/standalone/configuration/standalone.xml" tofile="${env.JBOSS_HOME}/standalone/configuration/${lra.coordinator.xml.filename}"></copy>
+                    <copy file="${env.JBOSS_HOME}/standalone/configuration/standalone.xml" tofile="${env.JBOSS_HOME}/standalone/configuration/${lra.participant.xml.filename}"></copy>
+                  </target>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-failsafe-plugin</artifactId>
             <configuration>
               <skipITs>false</skipITs>
               <systemProperties>
                 <arquillian.xml>arquillian-wildfly.xml</arquillian.xml>
+                <!-- qualifier used to identify the Wildfly cluster -->
+                <arquillian.group.qualifier>wildfly-cluster</arquillian.group.qualifier>
+                <!-- qualifier used to identify the Wildfly container where lra-coordinator will be deployed-->
+                <arquillian.lra.coordinator.container.qualifier>${lra.coordinator.container.qualifier}</arquillian.lra.coordinator.container.qualifier>
+                <!-- qualifier used to identify the Wildfly container where lra-participant will be deployed-->
+                <arquillian.lra.participant.container.qualifier>${lra.participant.container.qualifier}</arquillian.lra.participant.container.qualifier>
+                <!-- qualifier used to identify the lra-coordinator deployment-->
+                <arquillian.lra.coordinator.deployment.qualifier>${lra.coordinator.deployment.qualifier}</arquillian.lra.coordinator.deployment.qualifier>
+                <!-- qualifier used to identify the lra-participant deployment-->
+                <arquillian.lra.participant.deployment.qualifier>${lra.participant.deployment.qualifier}</arquillian.lra.participant.deployment.qualifier>
+                <!-- used to resolve the resteasy-client version -->
+                <version.resteasy-client>${version.resteasy-client}</version.resteasy-client>
```

Comment:
why do you need to redefine the same property?

Reply:
yes, I missed that this is setting system properties. I don't if it can be done other way. Dismiss this.